AoE2ScenarioParser/versions/DE/v1.54/effects.json

```diff
@@ -5332,5 +5332,145 @@
             "sound_name": "",
             "selected_object_ids": []
         }
+    },
+    "80": {
+        "name": "set_object_cost",
+        "attributes": [
+            "object_list_unit_id",
+            "source_player",
+            "quantity",
+            "tribute_list"
+        ],
+        "default_attributes": {
+            "effect_type": 80,
+            "tribute_list": 0,
+            "source_player": 1
+        }
+    },
+    "81": {
+        "name": "load_key_value",
+        "attributes": [
+            "variable",
+            "message",
+            "quantity"
+        ],
+        "default_attributes": {
+            "effect_type": 81,
+            "variable": 0
+        }
+    },
+    "82": {
+        "name": "store_key_value",
+        "attributes": [
+            "variable",
+            "message"
+        ],
+        "default_attributes": {
+            "effect_type": 82,
+            "variable": 0
+        }
+    },
+    "83": {
+        "name": "delete_key",
+        "attributes": [
+            "message"
+        ],
+        "default_attributes": {
+            "effect_type": 83
+        }
+    },
+    "84": {
+        "name": "change_technology_icon",
+        "attributes": [
+            "technology",
+            "source_player",
+            "quantity"
+        ],
+        "default_attributes": {
+            "effect_type": 84,
+            "quantity": 33,
+            "source_player": 1,
+            "technology": 16
+        }
+    },
+    "85": {
+        "name": "change_technology_hotkey",
+        "attributes": [
+            "technology",
+            "source_player",
+            "quantity"
+        ],
+        "default_attributes": {
+            "effect_type": 85,
+            "quantity": 18427,
+            "source_player": 1,
+            "technology": 16
+        }
+    },
+    "86": {
+        "name": "modify_variable_by_resource",
+        "attributes": [
+            "tribute_list",
+            "source_player",
+            "operation",
+            "variable"
+        ],
+        "default_attributes": {
+            "effect_type": 86,
+            "tribute_list": 0,
+            "source_player": 1,
+            "operation": 1,
+            "variable": 0
+        }
+    },
+    "87": {
+        "name": "modify_variable_by_attribute",
+        "attributes": [
+            "object_list_unit_id",
+            "source_player",
+            "operation",
+            "object_attributes",
+            "variable",
+            "message",
+            "string_id"
+        ],
+        "default_attributes": {
+            "effect_type": 87,
+            "source_player": 1,
+            "operation": 1,
+            "object_attributes": 0,
+            "variable": 0,
+            "string_id": 18427
```

Comment:
is this correct? did not see any string id here

Reply:
Yeah, I don't know why (it might be a bug?) But it is definitely set by the game

![image](https://github.com/user-attachments/assets/ba123702-506a-48ef-a2f2-1e30e2c583ea)

Reply:
@mardaravicius It is definitely a bug in the game. I'll remove it. Thx for the heads up